web/src/pages/apd/security-planning/SecurityPlanningReadOnly.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React, { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  selectSecurityInterfacePlan,
+  selectBusinessContinuityAndDisasterRecovery
+} from '../../../redux/selectors/apd.selectors';
+
+class SecuritySummary extends PureComponent {
```

Comment:
why is this a PureComponent?

Reply:
I was copying the [Executive Summary Read Only](https://github.com/Enterprise-CMCS/eAPD/blob/main/web/src/pages/apd/executive-summary/ExecutiveSummaryReadOnly.js) file.